app/styles/image-upload.css

```diff
@@ -59,7 +59,7 @@
 .image-preview {
   max-height: 500px;
   border-radius: 10px;
-  border: solid black;
+  border: solid var(--image-preview--border);
```

Comment:
On line 105, we are using the same color as a border, why not reuse it?
It's used for border, right?, we don't have to care for which component it is used